tsconfig.json

```diff
@@ -2,23 +2,26 @@
   "extends": "@tsconfig/node16/tsconfig.json",
   "files": [],
   "compilerOptions": {
-    "allowJs": true,
+    "allowJs": true
```

Comment:
`tsconfig.json` does not need to be valid JSON, we can have trailing commas and comments and stuff. There is still other non-json package tsconfig left after this. If we want to have them actual-JSON, I suggest to do that separate alongside a new linting enforcement?

Reply:
How is this a reason for blocking the entire PR?

Reply:
Also, I didn't do it. It was format on save from prettier. someone's gonna have to commit it or we'll all keep undoing.

Reply:
`tsconfig.json` is configured to be formatted as JSON, not JSONC (or whatever it is).  the automatic formatting is correct.